### DISCUSSION TITLE
TimeZoneEngine Thread Saftey after Initialization

### DISCUSSION THREAD

**Discussion by ivanthewebber** [RomanIakovlev/timeshape#138] [Q&A]
Is the TimeZoneEngine thread safe for concurrent queries after initialization?

Poking at the source code I think the answer is yes, but I think it makes sense to confirm and document it.

**Comment by RomanIakovlev** [comment]
Yeah I think it should be thread-safe, since most of the data is read-only after initialization, and the only mutable variables are method parameters and are therefore thread bound. If you want to make a PR to the README with this information, I'd gladly accept it.
